first-snap/python/test.yaml

```diff
@@ -0,0 +1,14 @@
+macos:
```

Comment:
I think 'test' instructions (possibly similar as build) are meant to be same for all languages on given OS, so we will only need one of those.